test/ComputeEngineComputationTest.java

```diff
@@ -20,7 +20,7 @@ public static void testComputeEngine(){
                     break;
                 }
                     if(tempValue < 0) {
-                        throw new IllegalArgumentException("nvalid input. Please enter a positive integer.");
```

Comment:
just a mispelling here but it looks good It wouldn't let me edit

Reply:
typo was fixed